tools: bump min node version to >=20.19

jason-ha opened the pull request:
20.19 is known to handle some eslint related issues better.
CI currently is using 20.19.0 and provides best option to verify what settings work.

jason-ha committed:
build: bump min node version to >=20.19
20.19 is known to handle some eslint related issues better.
CI currently is using 20.19.0 and provides best option to verify what settings work.

alexvy86 reviewed (approved):
Maybe bump it for build-tools as well? Note that it's also specified in some individual packages in that group. (Don't look at server/ or docs/ though 🙈 ).

jason-ha committed:
build: bump min node version to >=20.19
20.19 is known to handle some eslint related issues better.
CI currently is using 20.19.0 and provides best option to verify what settings work.

jason-ha commented:
> Maybe bump it for build-tools as well? Note that it's also specified in some individual packages in that group. (Don't look at server/ or docs/ though 🙈 ).

Well, my search and replace saw the horror. :(
